py3status/modules/mpd_status.py

```diff
@@ -156,34 +186,31 @@ def attr_getter(attr):
                 text = self.py3.safe_format(self.format, attr_getter=attr_getter)
 
         except socket.error:
-            text = "Failed to connect to mpd!"
-            state = None
+            text = self.mpd_no_conn
         except CommandError:
-            text = "Failed to authenticate to mpd!"
-            state = None
+            text = self.mpd_no_auth
```

Comment:
hmm,

Here people may want `mpd_no_auth = '[color=bad ...]'` so we should go through the formatter. we probably don't do that for the states though.

Should we just have these as `state_no_auth` and then just have them as alternative to `state_play` etc

It might be easiest to break this into parts.  I have only skimmed this PR so far.